manuscript/050_Its_Not_a_Test.md

```diff
@@ -68,12 +68,12 @@ The time has come to make a deal: I will establish some naming conventions for
 
 :   will be used instead of ‘passing test’
 
-From time to time, I will refer back to the ‘traditional’ terminology, because it is better established and you’ve probably heard some terms already and may wonder how it should be understood in context of thinking of tests as specification.
+From time to time I'll refer back to the ‘traditional’ terminology, because it is better established and because you’ve probably already heard some terms and may wonder how it should be understood in the context of thinking of tests as specification.
 
-From your experience, you may know paper or word specifications, written in plain English or other spoken language. Our specification is different than these specifications in at least few ways:
+From your experience, you may know paper or word specifications written in plain English or other spoken language. Our specification is different from these specifications in at least few ways:
 
-1.  it is not written fully up-front (more on this in the next chapters).
-2.  it is executable -- you can run it to see whether the code adheres to the specification or not.
-3.  it is written in source code rather than in spoken language -- which is both good (less room for misunderstanding -- source code is the most formal and structured way of writing specifications) and bad (great care must be taken to keep the specification readable).
+1.  It is not completely written up-front (more on this in the next chapters).
+2.  It is executable -- you can run it to see whether the code adheres to the specification or not.
+3.  It is written in source code rather than in spoken language -- which is both good, as there is less room for misunderstanding and bad, as great care must be taken to keep the specification readable.
```

Comment:
I like the re-structured sentence, however, I'd like to keep the information somehow that the reason source code leaves less room for understanding is that it is less ambiguous compared to spoken language.